test/integration/targets/inventory_vmware_vm_inventory/runme.sh

```diff
@@ -38,33 +40,42 @@ trap cleanup INT TERM EXIT
 echo "DEBUG: Using ${VCENTER_HOST} with username ${VMWARE_USERNAME} and password ${VMWARE_PASSWORD}"
 
 echo "Kill all previous instances"
-curl "http://${VCENTER_HOST}:5000/killall" > /dev/null 2>&1
+curl "http://${VCENTER_HOST}:${port}/killall" > /dev/null 2>&1
 
 echo "Start new VCSIM server"
-curl "http://${VCENTER_HOST}:5000/spawn?datacenter=1&cluster=1&folder=0" > /dev/null 2>&1
+curl "http://${VCENTER_HOST}:${port}/spawn?datacenter=1&cluster=1&folder=0" > /dev/null 2>&1
 
 echo "Debugging new instances"
-curl "http://${VCENTER_HOST}:5000/govc_find"
+curl "http://${VCENTER_HOST}:${port}/govc_find"
 
 # Get inventory
 ansible-inventory -i ${VMWARE_CONFIG} --list
 
+echo "Check if cache is working for inventory plugin"
+if [ ! -n "$(find "${inventory_cache}" -maxdepth 1 -name 'vmware_vm_inventory_*' -print -quit)" ]; then
+    echo "Cache directory not found. Please debug"
+    exit 1
+fi
+echo "Cache is working"
+
 # Get inventory using YAML
 ansible-inventory -i ${VMWARE_CONFIG} --list --yaml
 
 # Install TOML for --toml
-${PYTHON} -m pip install toml
+${PYTHON} -m pip freeze | grep toml > /dev/null 2>&1
+if [ $? -ne 0 ]; then
```

Comment:
I would prefer this do be done before, could we move this requirements in ./test/runner/requirements/integration.cloud.vcenter.txt ?